.github/workflows/develop.yml

```diff
@@ -25,5 +29,19 @@ jobs:
       - name: linter
         run: yarn lint
 
-      - name: test
-        run: yarn test
+      - name: build
+        run: yarn build-dist
+
+      - name: license report
+        run: yarn license-report
+
+      - name: compare artifact package version
+        run: ./.github/scripts/compare-artifact-package-versions.sh
+        env:
+          ARTIFACT_REGISTRY_PROVIDER: ${{ env.ARTIFACT_REGISTRY_PROVIDER }}
```

Comment:
I don't think you need this -- I believe any environment variables defined in the env file that `$GITHUB_ENV` points to will automatically be made available to all `run` jobs.